src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
missing class header comments - it should gives a brief idea of what the class is for.
e.g. `Represents a Person's contact` or `Contact for model` etc.